projects/batfish/src/main/java/org/batfish/main/Batfish.java

```diff
@@ -364,7 +359,8 @@ else if (containerDir != null) {
     */
    static List<Path> listAllFiles(Path configsPath) {
       List<Path> configFilePaths;
-      try (Stream<Path> allFiles = Files.walk(configsPath, FileVisitOption.FOLLOW_LINKS)) {
+      try (Stream<Path> allFiles = Files.walk(configsPath,
+            FileVisitOption.FOLLOW_LINKS)) {
```

Comment:
This is a place where 80 chars seems unnecessary.

Also, the break here is not semantic -- makes more sense to break at Files, at `.walk`, or at `configsPath`.